client/app/bundles/momentCard/components/MomentCardName.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from 'react';
+import css from './MomentCard.scss';
+
+type MomentCardNameState = {};
+
+type MomentCardNameProp = {
+  name: string;
+};
+
+export default class MomentCardName extends
+  React.Component<MomentCardNameProp, MomentCardNameState> {
```

Comment:
You can omit MomentCardViewersState as the second parameter (it's optional).

On the following lines, you can omit `props: MomentCardViewersProp` and `state: MomentCardViewersState` because they should already be defined as part of React.Component< MomentCardViewersProp>.